packages/unraid-shared/src/services/user-settings.ts

```
@@ -0,0 +1,142 @@
+
+/**
+ * A service for controlling exposed settings.
+ *
+ * This allows plugins to expose settings to users of the API without having to
+ * implement their own UI or api endpoints.
+ */
+@Injectable()
+export class UserSettingsService {
+  readonly settings = new Map<keyof UserSettings, FragmentOf>();
+
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)